app/templates/home.html

```diff
@@ -176,5 +177,22 @@ <h2 class="fw-bold text-success" id="thank-you-header">🎉 Thank You for Attend
     }
 </style>
 
+<script>
+    document.addEventListener("DOMContentLoaded", function () {
+        let container = document.getElementById("category-container");
+
+        function autoScroll() {
+            if (container.scrollLeft + container.clientWidth >= container.scrollWidth) {
+                container.scrollLeft = 0; // Reset scroll when reaching the end
+            } else {
+                container.scrollLeft += 2; // Adjust speed if needed
+            }
+        }
+
+        setInterval(autoScroll, 50); // Adjust speed by changing interval
```

Comment:
```suggestion
                container.scrollLeft += 1; // Adjust speed if needed
            }
        }

        setInterval(autoScroll, 16.6); // Adjust speed by changing interval
```
Smoother scroll, by targeting 60 frames-per-second (16.6666ms frame time).
For some reason, I could only get it to work with integer values for `scrollLeft`, so this ended up being slightly faster than before. An interval of 25 would be equivalent to the previous speed, albeit slightly choppier.

Reply:
it became way to choppy with a lower interval score so I kept that the same but decreased the speed.